lib/src/base-styles/_nx-typography.scss

```diff
@@ -38,44 +38,45 @@
 .nx-body {
   @include regular();
 
-  font-size: $nx-font-size-s;
+  color: $nx-text-color;
+  font-size: $nx-font-size;
```

Comment:
I'm noticing in the gallery that some code examples now scroll horizontally. I'm wondering if we should put the font size in the code examples back to 14 to avoid that.
![image](https://user-images.githubusercontent.com/3630091/84151231-0d364000-aa31-11ea-81eb-513835cf2deb.png)

Reply:
> should put the font size in the code examples back to 14 to avoid that.

Or maybe even 12

Reply:
> I'm noticing in the gallery that some code examples now scroll horizontally. I'm wondering if we should put the font size in the code examples back to 14 to avoid that.
> ![image](https://user-images.githubusercontent.com/3630091/84151231-0d364000-aa31-11ea-81eb-513835cf2deb.png)

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/40/commits/de9176b76764f09ae5e88dc22a9960c63f91627d

Reply:
> > should put the font size in the code examples back to 14 to avoid that.
> 
> Or maybe even 12

Note that not all of the examples use `GalleryExampleTile`. I'd prefer that we convert all of them first, and see where we're at before we drop down to 12px.